src/models/account.model.js

```diff
@@ -41,30 +41,36 @@ module.exports = class Account extends BaseModel {
     return account
   }
 
-  async save (authToken) {
+  async save (authToken, isDraft) {
     const dynamicsDal = new DynamicsDalService(authToken)
 
```

Comment:
The company number needs to be converted to upper case before it is is inserted / updated in Dynamics. The user can enter it in mixed case but when we persist it we should convert to upper case.